lib/actions-oclif/devices/supported.ts

```diff
@@ -29,10 +29,6 @@ interface FlagsDef {
 	verbose?: boolean;
 }
 
-interface DeviceTypeWithAliases extends SDK.DeviceType {
-	aliases?: string[];
```

Comment:
This was added to the SDK types themselves so it became unnecessary to extend them